src/ESPEasyRules.ino

```diff
@@ -248,16 +248,15 @@ void replace_EventValueN_Argv(String& line, const String& argString, unsigned in
   String eventvalue;
 
   eventvalue.reserve(16);
-  eventvalue  = F("%eventvalue");
-  eventvalue += argc;
+  eventvalue = F("%eventvalue");
+
+  if (argc != 0) {
+    eventvalue += argc;
+  }
```

Comment:
else argc =1;

Reply:
Nope, argc is given as function parameter and I extended the for loop calling the function.

Reply:
Ah, I see what you mean.....